new-docs/content/python/getting-started/roles/builtin_roles.md

```diff
@@ -42,15 +39,15 @@ relationships you can query, as well as `User.widget_roles` and `Widget.roles`
 to get the roles directly.
 
 {{% callout "Note" "blue" %}}
-  The names of the relationships we add to models are generated by the
-  following conventions:
-
-  * `Widget.users`: this relationship is always called `users`, regardless of
-    the actual user model name
-  * `User.widgets`: `Widget.__name__ + "s"`
-  * `Widget.roles`: this relationship is always called `roles`
-  * `User.widget_roles`: `Widget.__name__.lower() + "_roles"`
-{{% /callout %}}
+The names of the relationships we add to models are generated by the
```

Comment:
We're going to need to standardise on markdown formatters. Mine and Gabe's are incompatible too.

Reply:
Ugh yes forgot about this